src/content/en/tools/lighthouse/audits/manifest-contains-192px-icon.md

```diff
@@ -0,0 +1,53 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Manifest Contains Icons at Least 192px" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-21 #}
+{# wf_published_on: 2016-09-21 #}
+
+# Manifest Contains Icons at Least 192px  {: .page-title }
+
+## Why the audit is important {: #why }
+
+When a user adds your app to the homescreen, the mobile device needs an icon to
+display. That icon is specified in the `icons` array of the Web App Manifest.
+
+The presence of a 192-pixel icon ensures that your icon displays well on the
+largest Android devices. For smaller devices that need a smaller icon, Android
+can scale down the 192-pixel icon with reasonable accuracy. In other words,
+although you can provide smaller-sized icons in your Web App Manifest, it's
+unnecessary.
+
+## How to pass the audit {: #how }
+
+Add a 192-pixel icon to your Web App Manifest.
+
+    {
+      ...
+      "icons": [{
+        "src": "images/homescreen192.png",
+        "sizes": "192x192",
+        "type": "image/png"
+      }],
+      ...
+    }
+
+Check out [Manifest Exists](manifest-exists#how)
```

Comment:
If you wanted, you could put this into an [include file](https://developers.google.com/web/resources/widgets#includes), so that you only have to write it once, and when/if it's ever updated, you can simply update the include.